src/atoms/cc-input-number.js

```diff
@@ -187,6 +208,14 @@ export class CcInputNumber extends LitElement {
           </button>
         ` : ''}
       </div>
+
+      <div id=${this._uniqueHelpId}>
+        <slot name="help"></slot>
+      </div>
```

Comment:
Slots vs props: for error and help messages, I chose to use a slots because it allows devs to easily insert HTML into it (`<br>` mainly, but could be links also ?). I am not sure it's the best move as it means we have 2 empty `<div>` elements below fields. This does not cause any visual or accessibility issue but is it bad practice ?

Reply:
It's always the problem with "how can I know if a slot is used" :-(

I think it will be fine like this...